src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,26 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post: { id, title, body, user, comments } }) => (
+  <div className="PostInfo" key={id}>
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={user} />
```

Comment:
user can be undefined, so it's better to add the condition
```suggestion
        {user && <UserInfo user={user} />}
```